README.md

```diff
@@ -1,47 +1,115 @@
+the ruby-dev (Ubuntu) or ruby-devel (Fedora) package as well to build any
+native code for gems that are installed later. Installing these will also
+ensure your system's RubyGems package is installed. Depending on your OS, you
+may also need other packages to install/build gems, such as ruby-full, make,
+gcc, nodejs, build-essentials, or patch.
+
+Once Ruby, RubyGems, and any necessary native tools are installed, you are
+ready to install [Bundler] to manage the remaining RubyGem dependencies.
+Bundler is included in Ruby 2.6 and later as a default gem, so installing it
+may not be needed.
+
+Because we use [Bundler] to install specific versions of gems, it is not
+recommended to use an OS packaged version of gems other than what comes
+built-in. If you are using an OS packaged version of Ruby, it is __strongly__
+recommended to avoid `sudo` when installing additional gems, in order to avoid
+conflicting with your system's package-managed installation. Instead, you can
+specify a `GEM_HOME` directory for installing gems locally in your home
+directory. You can do this in your `$HOME/.bashrc` file or other appropriate
+place for your environment:
+
+```bash
+# in .bashrc
+export GEM_HOME=$HOME/.gem/ruby
+```
+
+With Ruby installed on your machine, you can install [Bundler] using the
+command below:
+
+```bash
+# not necessary in Ruby >2.6, since it is a default gem since 2.6
+gem install bundler
+```
+
+Next, use [Bundler] to install [Jekyll] and other dependencies needed to run
+the website (this command assumes your current working directory is your clone
+of this repository with the `master` branch checked out, because that's where
+the Gemfile dependency list exists).
 
-With Ruby installed on your machine, you can install [Bundler] using the command below:
+```bash
+bundle install
+```
 
-    gem install bundler
 
-Next, use [Bundler] to install [Jekyll] and other dependencies needed to run the website.
+## Basic Jekyll Build
 
-    git clone https://github.com/apache/accumulo-website
-    cd accumulo-website
-    bundle install
+The basic command to build the static site contents using Jekyll is:
+
+```bash
+bundle exec jekyll build
+```
 
-## Run the website locally
+The files generated by this command will be placed in `_site/`.
 
-Run the following command to run the website locally using Jekyll's embedded webserver:
 
-    cd accumulo-website
-    bundle exec jekyll serve -w
+## Running Locally for Testing/Development
+
+Rather than merely build the site contents, you can run a local webserver to
+view the contents in your browser. Jekyll provides a simple way to build and
+view the site contents with its own built-in webserver.
+
+Serving the site content this way may result in a view that is slightly
+different than a production site, because Jekyll's embedded webserver may have
+slightly different behavior than the webserver at apache.org, but it should be
+similar enough for testing most things.
```

Comment:
These two paragraphs are a bit wordy and could be shortened to:
"Jekyll provides a built-in webserver that is sufficient enough for testing but may result in a view that is slightly different than what actually gets deployed at apache.org.

Reply:
Will do.